README.md

```diff
@@ -6,52 +6,35 @@
 Visit the [ReaPack website](https://reapack.com/) for ready-to-use binaries,
 the user guide or the package upload tool.
 
-## Building from source
+## Building
 
-Clone the repository and submodules:
+Prerequisites:
 
+- [vcpkg](https://vcpkg.io/en/index.html)
+- [CMake](https://cmake.org/) 3.15 or newer
+- C++17 compiler
+- PHP (Linux and macOS)
+
+    # 1. Clone repository with submodules
     git clone --recursive --shallow-submodules https://github.com/cfillion/reapack.git
 
-### Prerequisites
+    # 2. Create build directory
+    mkdir build; cd build
 
-Software requirements:
+    # 3. Install dependencies and prepare files for build system
+    VCPKG_ROOT=path\to\vcpcg cmake -DCMAKE_BUILD_TYPE=Release ..
 
-- [CMake](https://cmake.org/) 3.15 or newer
-- C++17 compiler (MSVC on Windows)
-- PHP (Linux and macOS only)
+    # 4. Build
+    make
```

Comment:
Although it does by default on Unix-like systems, CMake does not always create a Makefile. For instance the recommended generator in this README is Ninja.

`cmake --build build` runs the correct command depending on which generator the user has selected.

Reply:
Ok, will revert to previous command